PLUB/Configuration/Manager/UserManager.swift

```diff
@@ -106,5 +111,30 @@ extension UserManager {
         }
       }
   }
+  
+  /// 최근 검색어목록에 키워드를 추가합니다
+  /// - Parameter keyword: 검색한 키워드
+  func addKeyword(keyword: String) {
+    let recentKeywordList = recentKeywordList ?? []
+    var filterList = recentKeywordList.filter { $0 != keyword }
+    filterList.insert(keyword, at: 0)
+    self.recentKeywordList = filterList
+  }
+  
+  /// 최근 검색어목록에 특정 인덱스에 위치한 키워드를 삭제합니다
+  /// - Parameter index: 삭제하고자하는 키워드의 위치
+  func removeKeyword(index: Int) {
+    guard var recentKeywordList = recentKeywordList,
+          index < recentKeywordList.count && 0 <= index else { return }
+    recentKeywordList.remove(at: index)
+    self.recentKeywordList = recentKeywordList
+  }
+  
+  /// 최근 검색어목록을 초기화합니다
+  func clearKeywordList() {
+    guard var recentKeywordList = recentKeywordList else { return }
+    recentKeywordList.removeAll()
+    self.recentKeywordList = recentKeywordList
+  }
```

Comment:
시도해보진 않았지만, 이렇게 줄였을 때 정상적으로 동작 될까요?
```suggestion
  func clearKeywordList() {
    self.recentKeywordList?.removeAll()
  }
```

Reply:
오 정상동작되네요 !! 저렇게 수정하도록 할게요 👍